### PR TITLE
Feature/docker compose new

### PR DESCRIPTION
**Pull request by Devansh3790**
Docker contain

### CODE REVIEW

File: src/dao/TweetDao.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import Tweet from "@db/models/Tweet";
```
Comment (amanprodigy) [inline src/dao/TweetDao.ts:1]: Why this change?

File: src/dao/UserDao.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import bcrypt from "bcrypt";
 import jwt from "jsonwebtoken";
 
-import User from "@db/models/User";
```
Comment (amanprodigy) [inline src/dao/UserDao.ts:4]: Same as above

File: start [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+npm start
```
Comment (amanprodigy) [inline start:3]: Is this file required for docker?

**Review by amanprodigy — APPROVED** [review]

File: src/dao/TweetDao.ts (hunk 1 above)
Comment (Devansh3790) [reply src/dao/TweetDao.ts:1]: Getting error when executing `npm start` from /start in the docker container.

Comment (Devansh3790) [reply src/dao/TweetDao.ts:1]: filenames you have given in small but here you using title case.

Comment (Devansh3790) [reply src/dao/TweetDao.ts:1]:
```
src/db/models/UserFollower.ts(8,18): error TS2307: Cannot find module '@db/models/User'.
src/db/models/tweet.ts(11,18): error TS2307: Cannot find module '@db/models/User'.
src/db/models/user.ts(14,19): error TS2307: Cannot find module '@db/models/Tweet'.
```

File: start (hunk 3 above)
Comment (Devansh3790) [reply start:3]: not required but added for future reference if we need to run more then one npm command.

File: src/dao/TweetDao.ts (hunk 1 above)
Comment (amanprodigy) [reply src/dao/TweetDao.ts:1]: Ok For this we should keep CamelCase of ModelNames and similar filenames. This will help us in keeping names as UserFollower.ts

Comment (Devansh3790) [reply src/dao/TweetDao.ts:1]: Do we need to change the camelCase or TitleCase? Will it be follow for the models files or throughout the app?

Comment (amanprodigy) [reply src/dao/TweetDao.ts:1]: Correct TitleCase
